src/main/msp/msp_stream.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+//
+// MSP Stream
+//
+
+#define MSP_STREAM_FLAG_STATUS                  (1 << 0) // get a list of the items that are being streamed
+#define MSP_STREAM_FLAG_BEGIN                   (1 << 1) // request to start streaming more items
+#define MSP_STREAM_FLAG_END                     (1 << 2) // request to stop streaming some items
+
+// items that can be stream include commands or values.
+#define MSP_STREAM_ENTRY_NONE                   (0 << 0)
+#define MSP_STREAM_ENTRY_COMMAND                (1 << 0)
+#define MSP_STREAM_ENTRY_VALUE                  (1 << 1)
```

Comment:
no implementation for MSP_STREAM_ENTRY_VALUE is done yet.  the idea is to allow a set of data items, each with an ID, to be requested.  see MSP_STREAM_VALUE_ID_* below.

I've not decided if i want to continue with this approach or just run only allowing existing command responses to be used.  my thinking behind it is if that if you only want a couple of items from an MSP command response it's wasteful to get a message and discard most of it.

I'm having thoughts about somehow tying this in with cli/get/set.